requirements/requirements.txt

```diff
@@ -5,7 +5,7 @@ onnx>=1.7.0
 python-dateutil
 ruamel.yaml
 scikit-learn
-setuptools==59.5.0
+setuptools==65.5.1
```

Comment:
out of my curiosity, why do we pin a specific version for `setuptools` instead of the latest version?

Reply:
I think people pinned it previously because newer version would cause issue that cannot load nemo. 
I upgraded it because our base container pytorch pinned this version and removed a vulnerability.

Reply:
gotcha. But not sure how much extra work we should do to fit the new version. I also observed a vulnerability here: https://github.com/NVIDIA/NeMo/security/dependabot/5

Reply:
ASAIK it was patched a while ago and has been reverted. 
I've tested it on my end and it worked fine. Asking more folks to try it out.